lib/Graph/Nodes.cpp

```diff
@@ -935,9 +935,12 @@ bool ExpNode::verify() const {
   const Node *parent = getResult().getNode();
   bool isValid =
       checkSameIsQuantized(getInput().getType(), getResult().getType(), parent);
+
   if (getInput().getType()->isQuantizedType()) {
-    return false;
+    isValid &= checkType(getInput(), getResult().getElementType(),
```

Comment:
So this check is for the same element type. But then below we also have `checkSameType()` which will also check the element types are the same in addition to check. So I think this isn't necessary/we can just remove the if here? In general these functions aren't named super nicely so it gets a little confusing...